### DISCUSSION TITLE
Shareable insights dashboards

### DISCUSSION THREAD

**Discussion by alantiller** [directus/directus#21092] [Feature Requests]
### Summary

The ability to create links to share insights panels with other people who do not have access to the application.

### Basic Example

Copy the code from the contents and apply it to the insights panel (adjustments would be necessary and a new public view would be needed to display just the insights panel.

### Motivation

For example have an uptime dashboard which I would love to share with others but currently have to use APIs and build its frontend meaning the insights panel is useless. It would be great if I could just share it like what can be done with tables.

### Detailed Design

The current design and operation of the sharing functionality would be a carbon copy of what is needed.

### Must Have:
- Ability to share individual insights board
- Ability to set timeouts passwords and view counts like the current system.

### Could Have:
- Ability to have an iFrame mode that allows for embedding

### Drawbacks

I don't think there are drawbacks as such but maybe you will need to consider table permissions for the tables and what data is being shown.

### Alternatives

I don't think there is a huge impact but I believe this will give a whole new set of use cases for the insights dashboards.

### Adoption Strategy

No there is no reason it should be a breaking change as it is just adding on top of existing functionality.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
